NitroxClient/MonoBehaviours/NitroxEntity.cs

```diff
@@ -86,6 +86,19 @@ public static void SetNewId(GameObject gameObject, NitroxId id)
             {
                 entity = gameObject.AddComponent<NitroxEntity>();
             }
+            if (gameObject.TryGetComponent(out UniqueIdentifier uniqueIdentifier) && uniqueIdentifier.id != id.ToString())
+            {
+                // To avoid unrequired error spams, we do the id setting manually
+                // If the current UID was already registered, we unregister it
+                if (!string.IsNullOrEmpty(uniqueIdentifier.id)
+                    && UniqueIdentifier.identifiers.TryGetValue(uniqueIdentifier.Id, out UniqueIdentifier registeredIdentifier) &&
+                    registeredIdentifier == uniqueIdentifier)
```

Comment:
```suggestion
                // To avoid useless error spams, we replace UWE ID with our own.
                if (!string.IsNullOrEmpty(uniqueIdentifier.id)
                    && UniqueIdentifier.identifiers.TryGetValue(uniqueIdentifier.Id, out UniqueIdentifier registeredIdentifier)
                    && registeredIdentifier == uniqueIdentifier)
                {
                    UniqueIdentifier.identifiers.Remove(uniqueIdentifier.id);
                }
                uniqueIdentifier.id = id.ToString();
                UniqueIdentifier.identifiers[id.ToString()] = uniqueIdentifier;
```

Reply:
Actually should be the other way around 😓 (with "&&" at the end of the line, then a new line)